coalib/output/Logging.py

```diff
@@ -4,25 +4,48 @@
 import logging
 import logging.config
 
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
 
-def configure_logging():
+
+def configure_logging(new_log_level=logging.INFO, incremental=False):
     """
     Configures the logging with hard coded dictionary.
+
+    If the incremental param is ``True``, all the log-handlers configuration
+    other than level will be ignored.
+    Thus, use it only when changing the log_level.
+    When changing ``log_level``, ``json-handler``, ``console-handler``,
+    and ``file-handler`` will have to exist.
```

Comment:
1. remove the comma before "and"
2. Maybe rephrase this to "To change log level with this function, it has to be called once with `incremental=False`"? That looks clearer to me^^